model/execution/DeliveryExecutionManagerService.php

```diff
@@ -285,7 +299,15 @@ public function adjustTimers(array $deliveryExecutions, $seconds)
 
                 $data = $deliveryMonitoringService->getData($deliveryExecution);
                 $data->updateData([DeliveryMonitoringService::REMAINING_TIME]);
+
                 $deliveryMonitoringService->save($data);
+
+                // log adjusted time of the de
```

Comment:
If you think this code block requires a comment maybe better extract it to private method.

Reply:
no, I guess variable name is clear